backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-2.adoc

```diff
@@ -0,0 +1,29 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-release-notes-1-2"]
+= OADP release notes
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: oadp-release-notes
+
+toc::[]
+
+The release notes for OpenShift API for Data Protection (OADP) 1.2 describe new features and enhancements, deprecated features, product recommendations, known issues, and resolved issues.
+
+include::modules/oadp-release-notes-1-2-3.adoc[leveloffset=+1]
+
+include::modules/oadp-release-notes-1-2-2.adoc[leveloffset=+1]
+
+include::modules/oadp-release-notes-1-2-1.adoc[leveloffset=+1]
+
+include::modules/oadp-release-notes-1-2-0.adoc[leveloffset=+1]
+include::modules/oadp-backing-up-dpa-configuration-1-2-0.adoc[leveloffset=+3]
+include::modules/oadp-upgrading-oadp-operator-1-2-0.adoc[leveloffset=+3]
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc#migration-configuring-aws-s3_installing-oadp-aws[Configuring Amazon Web Services].
+* xref:../../../backup_and_restore/application_backup_and_restore/installing/oadp-using-data-mover-for-csi-snapshots-doc.adoc#oadp-using-data-mover-for-csi-snapshots-doc[Using Data Mover for CSI snapshots].
```

Comment:
Not blocking merge for this, but just noticed periods on the ends of these bullets that could be removed in a later PR.